lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -1271,6 +1276,18 @@ ChangeResult TypeAnalyzer::visitAtenSoftmaxIntOp(
   return getLatticeElement(op.getResult()).join(knowledge);
 }
 
+ChangeResult TypeAnalyzer::visitAtenLogSoftmaxIntOp(
```

Comment:
Shall we reuse `TypeAnalyzer::visitAtenSoftmaxIntOp`?  Here is a similar example using template to refactor the code `https://github.com/llvm/torch-mlir/blob/127c7d8e27ede211af77392b5277ba06fb515fb7/lib/Dialect/Torch/Transforms/RefineTypes.cpp#L326`

Reply:
Done.